src/components/FieldText.js

```diff
@@ -16,6 +16,18 @@ const mapError = ({
     }
     : { ...input, ...props });
 
+const required = value => (value ? undefined : 'Required');
```

Comment:
They could, but in this case they will be used through the exported components, so we wont need them directly.